### PR TITLE
update context data excel file

### PR DESCRIPTION
**Pull request by markushal**
I updated the excel file. The contents of the file seem to be OK, the remaining bugs in the display of context data can be fixed in the python code.

### CODE REVIEW

**Comment by markushal** [comment]
@joAschauer Sorry, I did not clear the cache before testing. It seems that test are failing and the the structure of the excel file changed. I have to check this before you can review the PR.

**Comment by markushal** [comment]
@joAschauer OK, now the tests are passing. I guess we can merge the PR now.

File: app/tab_certification_schemes.py [hunk 1]
```diff
@@ -21,7 +21,6 @@ def _render_scheme_info(context_data, scheme_name):
         st.markdown(
             f"- **Relation to other standards:** {data['relation_to_other_standards']}"
         )
-        st.markdown(f"- **Geographic scope:** {data['geographic_scope']}")
```
Comment (joAschauer) [inline app/tab_certification_schemes.py:24]: why did you remove this?
